src/skip.js

```diff
@@ -75,7 +75,7 @@ class Skip extends BasePlugin {
   _setOutroData(outro: SkipPoint): void {
     if (typeof outro?.startTime === 'number') {
       if (typeof outro?.endTime !== 'number' || outro?.endTime === -1) {
-        outro.endTime = this.player.duration;
+        outro.endTime = this.player.duration - 1;
```

Comment:
I would try to use `isPlaybackEnded` prop from store instead of this workaround

Reply:
I’ve tried both options, it will solve the problem but just partially, what happens is that the button flickers few times before it disappears

Reply:
what causes it to flicker?

Reply:
the current time  - even you seek to the end it take it a few ms to be equal to duration
log example :
## on {duration: 581.009704, currentTime: 580.909704, endTime: 581.006} 
## off {duration: 581.009704, currentTime: 581.009704, endTime: 581.006} 

so i mean even we go with the one of the above two options the -1 is prevent the flashing